examples/post_to_mqtt.py

```diff
@@ -0,0 +1,91 @@
+#!/bin/python3
+# -*- coding: utf-8 -*-
+"""
+Get data from sensors and post it to specified mqtt channel in json-format
+
+Requires:
+    Ruuvitag sensor - pip3 install --user ruuvitag-sensor
+    Paho MQTT       - pip3 install --user paho-mqtt
+"""
```

Comment:
You could add an example command here how to execute this file e.g. `post_to_mqtt.py --mac AA:BB:CC:DD:EE:FF --broker xxxx`

Reply:
Thanks for the comments, it made it better. Didn't remember to use linter, good point. I added the example, along with link to Ansible automated version on how I handle sauna temperature at home. It set's the systemd file to keep the service going, along with dedicated user.